cmd/blox_build.go

```diff
@@ -24,11 +25,16 @@ var buildCmd = &cobra.Command{
 	Use:   "build",
 	Short: "Validate and build your data",
 	Run: func(cmd *cobra.Command, args []string) {
-		database, err := cuedb.NewDatabase()
+		// This can happen at a global Cobra level, if I knew how
+		config, err := cuedb.NewRuntime()
```

Comment:
Separate runtime for config instead of piggy backing on the database

Reply:
this will get cleaned up in my forthcoming pr that moves config out on its own